llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -2141,6 +2141,7 @@ gentbl(
             ":Core",
             ":IPO",
             ":MC",
+            ":Passes",
```

Comment:
I think this is an upstream layering violation -- can we wait to get it fixed?

I've sent a note to some folks about it already.

Reply:
Sure, although I think we could also merge this with a TODO and then delete it again when it's fixed? It seems good to keep these up to date.

Reply:
You're right.

Reply:
Good because I now get an email every 15 minutes that the integration branch is broken :-D

Reply:
```suggestion
            ":Passes",  # TODO(chandlerc): Likely a layering violation.
```

Reply:
Lol GitHub.... The easiest way to edit in the UI is to create a suggested edit myself. If I go to edit the file it just pulls up the whole thing